site/content/contribute/why-contribute/_index.md

```diff
@@ -95,6 +100,7 @@ Thank you for your enthusiasm! You can act on feature ideas in a few ways:
 4. Discuss your idea with our community in the [Feature Proposals channel](https://community.mattermost.com/core/channels/feature-ideas).
 5. Build an app! Mattermost has a rich framework full of tools to help you add the features you want that don’t quite work as core additions to Mattermost.
     - [Webhooks]({{< ref "/integrate/webhooks" >}}) are the easiest type of app to create and enable you to integrate with the vast majority of modern web tools.
+  
```

Comment:
was this intended?

Reply:
Yes - I've added a bit of space between bullet points and sub-bullets to make them a bit easier to scan/read.